cypress/cypress/actions/signup.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable max-len */
+import SignupPage from '../integration/pom/SignupPage';
+
+const signup = (userType, {
+  phoneNumber = '', password = '', passwordConfirm = password, firstName = '', lastName = '', email = '', carRegistrationNumber = '', carManufacturer = '', carModel = '', imgPath = '',
```

Comment:
Yo can avoid lines like this (very big lines)

Reply:
yeah,  should have put them on separate lines 😄